arch/arm/configs/bcm2709_defconfig

```diff
@@ -725,6 +725,7 @@ CONFIG_RPI_POE_POWER=m
 CONFIG_BATTERY_DS2760=m
 CONFIG_BATTERY_MAX17040=m
 CONFIG_BATTERY_GAUGE_LTC2941=m
+CONFIG_CHARGER_GPIO=m
```

Comment:
All of these additions need to go before CONFIG_BATTERY_GAUGE_LTC2941. When creating patches like this, use `make ... xxx_defconfig`, `make ... savedefconfig` and compare the output file (`defconfig`) with the `xxx_defconfig` file in questions. They should be identical.